PDFOptimizer/Release_Notes.html

```diff
@@ -18,6 +18,20 @@
         <h1>Release Notes, Datalogics PDF Optimizer</h1>
   
 	<div class="entry-content">
+<p><strong>Version 3.3.0</strong> (August 8, 2023)<br/>
+<p>New Features:</p>
+<ul>
+  <li>Added PDF/X-6 Output Support</li>
+  <li>Added support for a Linux ARM version of PDF Optimizer</li>
+</ul>
+<p>Problem Corrections:</p>
+<ul>
+  <li>Fixed a crash generating reports for certain input</li>
```

Comment:
Based on the date of PDFOPT-217, this appears to have actually been fixed in v3.1.1